corehq/apps/app_manager/views/forms.py

```diff
@@ -546,12 +547,19 @@ def get_form_view_context_and_template(request, domain, form, langs, messages=me
         form.get_unique_id()
         app.save()
 
+    allow_usercase = (domain_has_privilege(request.domain, privileges.USER_CASE)
+                      and not toggles.USER_TESTING_SIMPLIFY.enabled(request.domain))
+    valid_index_names = DEFAULT_CASE_INDEX_IDENTIFIERS.values()
+    if allow_usercase and USERCASE_PREFIX[-1] == "/":
```

Comment:
The usercase prefix used to be `user:`, so I wanted to not add it to the prefixes if that was the case. Although, odds are good it was changed to `user/` to resemble `parent/` etc and isn't likely to change back... @kaapstorm do you think this check is worthwhile, or is `user/` set in stone at this point?